test/built-ins/ThrowTypeError/unique-per-realm-unmapped-args.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2016 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-%throwtypeerror%
+description: >
+  %ThrowTypeError% is defined once for each realm.
+info: >
+  %ThrowTypeError% ( )
+
+  The %ThrowTypeError% intrinsic is an anonymous built-in function
+  object that is defined once for each realm.
+---*/
+
+var ThrowTypeError = Object.getOwnPropertyDescriptor(function(){ "use strict"; return arguments; }(), "callee").get;
+
+// Test with unmapped arguments object from strict-mode function.
```

Comment:
this comment - as in the other unique-per-realm tests - could be part of the description. It is even ok to have more than a single line description if necessary.